README.md

```diff
@@ -1,46 +1,148 @@
 
 ### Stakeholder Backlog
-
-1. all unassigned issues are reviewed by the product owner
-2. all assigned issues have to be completed by the assignee and unassigned once ready for review
-
-This loop continues until the [product owner](#product-owner) moves it into the product backlog.
+The goal of the [Stakeholder Backlog] is to collect issues and ideas (notes) from all Stakeholders/ users/ contributors/ etc..
+Issues and ideas can be unprioritzed, unsorted and even not well defined.
+The [Product Owner] reviews them regularly and decides, which of these need more attention, need more details and/ or need to be worked on (prioritized).
+
+  1. all issues start in the Stakeholder Backlog
+  1. all new and unassigned issues are reviewed by the [product owner] ([PO])
+  1. the product owner decides if an issue is well defined and needs to be prioritized
+     1. if an issue is well defined, the [PO] sets the label to `triage/accepted`
+     1. if an issue is not well defined, the [PO] assigns it to a contributor e.g. for decscribing it better or get more details
+        1. This loop continues until the [PO] sets the label `triage/accepted` and leaves it unassigned
+     1. if an issue is well defined `triage/accepted` and is prioritized by the [PO], the [PO] moves it to the [Product Backlog]
+  1. all assigned issues have to be completed by the assignee and unassigned once ready for review
 
 ### Product Backlog
-
-Issues in the product backlog are good to move to `Sprint Backlog` during the
-Sprint planning. They are *not assigned* to any person.
-
-## Sprint planning / review meeting
-
-### Prior to the meeting
-
-Go through all the issues in the `Done` column and +1 or -1 any issue.
-
-### During the meeting
+The goal of the [Product Backlog] is to have a collection of well defined (`triage/accepted`) and ready to be worked on issues.
+ - Issues in the [Product Backlog] are prioritized and need to be worked on in the next sprints.
+ - During the [Sprint Planning] these issues can be moved to the [Sprint Backlog].
+ - The [Product Backlog] holds only issues;
+these issues must be `triage/accepted`, but they don't need to be "assigned" to any person yet.
+
+### In Progress
+The goal of [In Progress] is to see, which issues are currently worked on.
+ - During the [Sprint Planning] issues are assigned and moved to the [Sprint Backlog].
+ - All issues in the [Sprint Backlog] while be worked on in during the [Sprint].
+ - When an assignee actively works on one or more of his/her/* issues, the assignee moves the issue to [In Progress].
+
+### Done
+The goal of [Done] is to collect all the solved issues.
+ - Issues here are ready to be tested/ checked/ approved by the Stakeholder (creator of the issue in general). 
+ - Solved is defined as: the assignee took care of all the mentioned acceptance criterias during the sprint.
+ - The assignee defines solved and moves the issue from [In Progress] to [Done].
```

Comment:
>   - The assignee defines solved and moves the issue from [In Progress] to [Done].

not sure what you mean by "assignee defines solved"

Reply:
Maybe a lack of a better wording.
What I like to say:
An issue is a problem that should be solved, or creates an MVP, or has acceptance criteria.
An assignee works on these things and "solves" them, or creates results, that fit the criteria.
When this happened, and the assignee decides this, the issue can move on column further, to the Done column.
Bascically: done = solved (from the assigne's view)
If the issue is done/ solved in the stakeholders view, will be the next step.

Reply:
I guess and hopem @HumairAK 's `thumbs up`is an OK, and I will call this resolved.